### PR TITLE
Fixes #69: Adapter for Google Identity Toolkit v3 

### LINKED ISSUE #69: Adaper for `www.googleapis.com/identitytoolkit`

### PR DESCRIPTION
**Pull request by baltpeter**
This is based on #59, which needs to be merged first.

### CODE REVIEW

File: src/adapters/google.ts [hunk 1]
```diff
@@ -450,4 +488,115 @@ export const adapters: Adapter[] = [
...
+            { function: 'parseJson', input: 'body', output: 'res.body' },
+            { function: 'getProperty', input: 'header', output: 'res.header', options: { path: '$' } },
+        ],
+        containedDataPaths: {
+            ...containedDataPathsIdentityToolkitHeaders,
+
+            otherIdentifiers: {
+                context: 'body',
+                path: 'idToken',
+                reasoning: 'google/idToken.md',
```
Comment (zner0L) [inline src/adapters/google.ts:510]: Shouldn’t we decode the JWT and extract the info such as the id here? Or do you think these tokens are too diverse to be usefully decoced?

**Comment by zner0L** [comment]
Created #79 to track JWT decoding and will hold merging until this is closed.

**Comment by baltpeter** [comment]
This is now based on #81, which needs to be merged first.

**Review by zner0L — APPROVED** [review]
Thank you for the changes!

**Pull request merged**